noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -2153,10 +2315,15 @@
                 ]]>
             </warnings>
         </queries>
-    </rule>
-
-    <rule>
-        <title>Period containment of registry entries</title>
+    </test>
+
+    <test id="LJT2">
+        <title>Period containment of Registry Entries</title>
+        <description>
+            Tests whether the Registry Entries' created and finalized dates are within
```

Comment:
`Tests whether the created and finalized dates of all registry entries in loependeJournal.xml are within the archival period specified in arkivuttrekk.xml, and whether the same registry entries (same systemIDs) can be found in arkivstruktur.xml`